src/events/cd-event-sessions.vue

```diff
@@ -62,12 +51,12 @@
         event: null,
         children: [{ value: {}, id: uuid() }],
         valid: true,
+        validPhone: true,
         user: {},
         phone: '',
         submitted: false,
       };
     },
-    // needs tests for these bois
     computed: {
       showPhone() {
```

Comment:
What's the idea behind showPhone? Is it to display only for parents rather than o13 ?
The current condition basically forces it to anyone which doesn't have a phone number, which I don't understand

Reply:
So it's more a retro-compatibility thingy more than anything else for previous parents (returning users) for which the phone number is missing? cc @conoramurphy 
In which case, it can be simplified into `return !!this.user.phone`

Reply:
yea exacty